Remotion/Web/UnitTests/Core/UI/Controls/CommandTests/CommandTestHelper.cs

```diff
@@ -197,10 +187,10 @@ public Command CreateEventCommand (IWebSecurityAdapter webSecurityAdapter = null
       return command;
     }
 
-    public Command CreateEventCommandAsPartialMock ()
+    public Mock<Command> CreateEventCommandAsPartialMock ()
     {
-      Command command = _mocks.PartialMock<Command> (CommandType.Event, (IWebSecurityAdapter) null, (IWxeSecurityAdapter) null);
-      InitializeEventCommand (command);
+      var command = new Mock<Command> (CommandType.Event, (IWebSecurityAdapter) null, (IWxeSecurityAdapter) null) { CallBase = true };
```

Comment:
There is no `.CallOriginalMethod (OriginalCallOptions.NoExpectation)` here, are you sure that `{ CallBase = true }` must be set?

Reply:
yes `{ CallBase = true }` must be set, because in `Rhino.Mocks` the mock was created as a `PartialMock`